src/Tribe/Tickets.php

```diff
@@ -1,2299 +1,2345 @@
 <?php
+/**
+ * Class with the API definition and common functionality
+ * for Tribe Tickets Pro. Providers for this functionality need to
+ * extend this class. For a functional example of how this works
+ * see Tribe WooTickets.
+ *
+ * The relationship between orders, attendees and event posts is
+ * maintained through post meta fields set for the attendee object.
+ * Implementing classes are expected to provide the following class
+ * constants detailing those meta keys:
+ *
+ *     ATTENDEE_ORDER_KEY
+ *     ATTENDEE_EVENT_KEY
+ *     ATTENDEE_PRODUCT_KEY
+ *
+ * The post type name used for the attendee object should also be
+ * made available via:
+ *
+ *     ATTENDEE_OBJECT
+ *
+ *
+ * @since  4.5.0.1 Due to a fatal between Event Ticket Plus extending commerces and this class,
+ *                 we changed this from an Abstract to a normal parent Class
+ */
+class Tribe__Tickets__Tickets {
 
-if ( ! class_exists( 'Tribe__Tickets__Tickets' ) ) {
```

Comment:
I know it kind of sucks, but can you retain this nesting to reduce the conflicts we'll see when merging into Tribe Commerce? A change on every line will be a bastard to sift through. We can reduce the nesting in _that_ branch instead.

Reply:
hahaha Sounds good. I can see how that would be a problem.